seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcMysqlSplitIT.java

```diff
@@ -0,0 +1,500 @@
+                                new byte[] {byteArr},
+                                new byte[] {byteArr, byteArr},
+                                new byte[] {byteArr, byteArr, byteArr, byteArr},
+                                new byte[] {
+                                    byteArr, byteArr, byteArr, byteArr, byteArr, byteArr, byteArr,
+                                    byteArr
+                                },
+                                i % 2 == 0 ? Boolean.TRUE : Boolean.FALSE,
+                                i,
+                                i,
+                                i,
+                                i,
+                                i,
+                                i,
+                                i,
+                                i,
+                                i,
+                                Long.parseLong(i + ""),
+                                Long.parseLong(i + ""),
+                                Long.parseLong(i + ""),
+                                BigDecimal.valueOf(i, 0),
+                                BigDecimal.valueOf(i, 18),
+                                BigDecimal.valueOf(i, 18),
+                                Float.parseFloat(i + ".1"),
+                                Float.parseFloat(i + ".1"),
+                                Double.parseDouble(i + ".1"),
+                                Double.parseDouble(i + ".1"),
+                                "f",
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("f1_%s", i),
+                                String.format("{\"aa\":\"bb_%s\"}", i),
+                                String.format("f1_%s", i),
+                                Date.valueOf(currentDate),
+                                Timestamp.valueOf(LocalDateTime.now()),
+                                new Timestamp(System.currentTimeMillis()),
+                                "test".getBytes(),
+                                "test".getBytes(),
+                                "test".getBytes(),
+                                "test".getBytes(),
+                                "test".getBytes(),
+                                "f".getBytes(),
+                                bigintValue.add(BigDecimal.valueOf(i)),
+                                decimalValue.add(BigDecimal.valueOf(i)),
+                                decimalValue.add(BigDecimal.valueOf(i)),
+                            });
+            rows.add(row);
+        }
+
+        return Pair.of(fieldNames, rows);
+    }
+
+    static JdbcUrlUtil.UrlInfo mysqlUrlInfo =
+            JdbcUrlUtil.getUrlInfo(
+                    String.format("jdbc:mysql://localhost:%s/auto?useSSL=false", MYSQL_PORT));
+
+    @TestTemplate
+    public void testSplit(TestContainer container) throws Exception, InterruptedException {
```

Comment:
```suggestion
    @Test
    public void testSplit() {
```
Please use `@Test` not `@TestTemplate`